docs/resources/slo.md

```diff
@@ -145,67 +145,95 @@ resource "nobl9_slo" "this" {
+- `display_name` (String) Name to be displayed.
+- `name` (String) Objective's name. This field is computed if not provided.
+- `op` (String) Type of logical operation.
+- `primary` (Boolean) Is objective marked as primary.
+- `raw_metric` (Block Set) Raw data is used to compare objective values. (see [below for nested schema](#nestedblock--objective--raw_metric))
+- `time_slice_target` (Number) Designated value for slice.
 
-<a id="nestedblock--objective"></a>
-### Nested Schema for `objective`
+<a id="nestedblock--objective--composite"></a>
+### Nested Schema for `objective.composite`
 
 Required:
 
-- `display_name` (String) Name to be displayed
-- `target` (Number) Designated value
-- `value` (Number) Value
+- `max_delay` (String) Maximum time for your composite SLO to wait for data from objectives.
 
 Optional:
 
-- `count_metrics` (Block Set) Compares two time series, calculating the ratio of either good or bad values to the total number of values. Fill either the 'good' or 'bad' series, but not both. (see [below for nested schema](#nestedblock--objective--count_metrics))
-- `name` (String) Objective's name. This field is computed if not provided.
-- `op` (String) Type of logical operation
-- `primary` (Boolean) Is objective marked as primary.
-- `raw_metric` (Block Set) Raw data is used to compare objective values. (see [below for nested schema](#nestedblock--objective--raw_metric))
-- `time_slice_target` (Number) Designated value for slice
+- `components` (Block Set) Objectives to be assembled in your composite SLO. (see [below for nested schema](#nestedblock--objective--composite--components))
+
+<a id="nestedblock--objective--composite--components"></a>
+### Nested Schema for `objective.composite.components`
+
+Optional:
+
+- `objectives` (Block Set) An additional nesting for the components of your composite SLO. (see [below for nested schema](#nestedblock--objective--composite--components--objectives))
+
+<a id="nestedblock--objective--composite--components--objectives"></a>
+### Nested Schema for `objective.composite.components.objectives`
+
+Optional:
+
+- `composite_objective` (Block List) Your composite SLO component. (see [below for nested schema](#nestedblock--objective--composite--components--objectives--composite_objective))
+
+<a id="nestedblock--objective--composite--components--objectives--composite_objective"></a>
+### Nested Schema for `objective.composite.components.objectives.composite_objective`
+
+Required:
+
+- `objective` (String) SLO objective name.
+- `project` (String) Project name.
+- `slo` (String) SLO name.
+- `weight` (Number) Weights determine each component’s contribution to the composite SLO.
+- `when_delayed` (String) Indicator of how to interpret data received after max_delay is over.
```

Comment:
The data might never been received even after the delay - so the story is not about delayed data it's about missing data

```suggestion
- `when_delayed` (String) An indicator of how to interpret the component not receiving any data before `max_delay' elapses
```

Reply:
Thanks, I missed that. Consulted it with tech writers again and now it's `Defines how to treat missing component data on 'max_delay' expiry.`. Is it good in your opinion?